packages/itwinui-css/src/tabs/default.scss

```diff
@@ -82,6 +82,9 @@
 
     .iui-tab-description {
       display: block;
+      @supports (-webkit-line-clamp: 1) {
+        display: -webkit-box; /* stylelint-disable-line */
```

Comment:
Is this needed? The line clamp mixin should already set it

Reply:
It is needed.. We have `display: none` which needs to be overriden and its happening here :/

Reply:
I thought so too after seeing >90% support.. But kinda left same as in the mixin :/

Reply:
changed